### PR TITLE
User story 26

### PR DESCRIPTION
**Pull request by ggerdsen**
User Story 26, Registered users can check out

As a registered user
When I add items to my cart
And I visit my cart
I see a button or link indicating that I can check out
And I click the button or link to check out and fill out order info and click create order
An order is created in the system, which has a status of "pending"
That order is associated with my user
I am taken to my orders page ("/profile/orders")
I see a flash message telling me my order was created
I see my new order listed on my profile orders page
My cart is now empty

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -50,6 +50,7 @@
   get "/profile/password", to: 'users#edit_password'
   patch '/profile', to: 'users#update'
   put '/profile', to: 'users#update_password'
+  get "profile/orders", to: "orders#index"
```
Comment (JBelta) [inline config/routes.rb:53]: Still going to look into namespace/ resouces. Maybe wait until re-iteration at the end of the project.

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -3,23 +3,33 @@ class OrdersController <ApplicationController
...
+    @user = User.find(current_user.id)
+  end
 
   def show
     @order = Order.find(params[:id])
   end
 
   def create
-    order = Order.create(order_params)
+    order = current_user.orders.create(order_params)
```
Comment (Callbritton) [inline app/controllers/orders_controller.rb:16]: This is an interesting solution!

File: db/schema.rb [hunk 3]
```diff
@@ -22,6 +22,7 @@
     t.integer "quantity"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "status"
```
Comment (JBelta) [inline db/schema.rb:26]: have the status be determined by "quantity > 0" ? have that effect our order status?

File: db/schema.rb [hunk 4]
```diff
@@ -58,6 +58,8 @@
     t.integer "zip"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.bigint "user_id"
```
Comment (JBelta) [inline db/schema.rb:62]: shweet

**Pull request merged**